PhysicsTools/NanoAOD/python/triggerObjects_cff.py

```diff
@@ -219,7 +219,22 @@
 )
 
 from PhysicsTools.PatUtils.L1ECALPrefiringWeightProducer_cff import prefiringweight
-run2_HLTconditions_2016.toModify(prefiringweight, DataEra = cms.string("2016BtoH"))
+from PhysicsTools.NanoAOD.nano_eras_cff import *
+from PhysicsTools.NanoAOD.common_cff import *
+#Next line will be updated once we get UL2016 maps
+run2_jme_2016.toModify( prefiringweight, DataEra = cms.string("2016BtoH"))
+#Next line is for UL2017 maps 
+run2_jme_2017.toModify( prefiringweight, DataEra = cms.string("UL2017BtoF"))
```

Comment:
for the 2018 (UL) the default is what is in the fillDescription i.e.   
desc.add<std::string>("DataEra", "2017BtoF"); 
is this correct ?

Reply:
Yes. But there's no prefiring in 2018 so that shouldn't be used. Currently the prefiring weights are not even computed/saved for 2018.

Reply:
ok.
[mc106Xul17_doc.html#L1PreFiringWeight](indeed the https://cms-nanoaod-integration.web.cern.ch/integration/master-cmsswmaster/mc106Xul17_doc.html#L1PreFiringWeight)
is not stored in the 2018
[mc106Xul18_doc.html](https://cms-nanoaod-integration.web.cern.ch/integration/master-cmsswmaster/mc106Xul18_doc.html)